web/yarn.lock

```diff
@@ -1266,22 +1266,22 @@
     "tslib" "^2.1.0"
     "xmlhttprequest" "1.8.0"
 
-"@firebase/auth-interop-types@0.1.6":
-  "integrity" "sha512-etIi92fW3CctsmR9e3sYM3Uqnoq861M0Id9mdOPF6PWIg38BXL5k4upCNBggGUpLIS0H1grMOvy/wn1xymwe2g=="
-  "resolved" "https://registry.npmjs.org/@firebase/auth-interop-types/-/auth-interop-types-0.1.6.tgz"
-  "version" "0.1.6"
+"@firebase/auth-interop-types@0.1.5":
```

Comment:
Seems like you downgraded/removed most of the things here. Any reason for that?

Reply:
this is autogenned, I didn't do anything
it's probably just stale relative to master so it's showing as "removed" when in actually its just that master has gone ahead of this branch. Should be fixed by rebase